tests/testthat/test-conditional_formatting.R

```diff
@@ -1,259 +1,394 @@
-test_that("conditional formatting", {
 
+negStyle <- create_dxfs_style(font_color = c(rgb = "FF9C0006"), bgFill = c(rgb = "FFFFC7CE"))
+posStyle <- create_dxfs_style(font_color = c(rgb = "FF006100"), bgFill = c(rgb = "FFC6EFCE"))
+
+my_workbook <- function() {
+  # Make the workbook each time so wbWorkbook$debug() is easier to access within
+  # each test
   wb <- wb_workbook()
-  wb$add_worksheet("cellIs")
+  wb$styles_mgr$styles$dxfs <- c(wb$styles_mgr$styles$dxfs, c(negStyle, posStyle))
+  invisible(wb)
+}
 
-  negStyle <- create_dxfs_style(font_color = c(rgb = "FF9C0006"), bgFill = c(rgb = "FFFFC7CE"))
-  posStyle <- create_dxfs_style(font_color = c(rgb = "FF006100"), bgFill = c(rgb = "FFC6EFCE"))
+expect_save <- function(wb) {
+  # Quick function to check original saving.
+  assert_workbook(wb)
+  path <- temp_xlsx()
 
-  wb$styles_mgr$styles$dxfs <- c(wb$styles_mgr$styles$dxfs,
-                                 c(negStyle, posStyle)
-  )
+  expect_silent(wb_save(wb, path))
```

Comment:
maybe needs `testthat::`

Reply:
Eh, anything within `tests/testthat/` can probably safely assume `testthat` is loaded.

Reply:
dunno, but it's a function using testthat wrappers and lintr complains. Therefore my idea, but we might as well: not care ;)

Reply:
Fair enough.  I was just trying to be a little lazy.